README.md

```diff
@@ -382,6 +386,13 @@ Simple static plots are generated from the outputs of Counter and DESeq2. These
 - **sample_avg_scatter_by_dge**: A scatter plot comparing all sample groups, with significantly different genes highlighted.
 - **sample_avg_scatter_by_dge_class**: A scatter plot comparing all sample groups, with classes and significantly different genes highlighted.
```

Comment:
Perhaps say "....with classes highlighted for differentially expressed small RNAs based on P value cutoff."   More generally, instead of "genes" (as in line 386 above as well), use "small RNAs" (or we could use "features"), and instead of "significantly different", let's use "differentially expressed".

Reply:
That sounds much better. I've updated README.md and Plotter's helpstring with these changes